benchmark.sh

```diff
@@ -30,8 +34,4 @@ printf "\nmemsweep:\n">> results.txt
 sh memsweep.sh | grep "seconds" >> results.txt
 
 printf "\nHardware:\n" >> results.txt
-sudo lshw -short >> results.txt
-
-
-rm linpack
```

Comment:
`remote_benchmark.sh` uses git to upload `results.txt` and the binaries are in `.gitignore`.